src/Endpoints/ZakenEndpoint.php

```diff
@@ -51,7 +51,7 @@ protected function buildEntity($data): Entity
         $class = $this->entityClass;
         $zaak = new $class($data, $this->client::CALLABLE_NAME, $this->client::CLIENT_NAME);
 
-        $statusToelichting = $zaak->status->statustoelichting ?? '';
+        $statusToelichting = $zaak->status->statustype->statusExplanation();
```

Comment:
@sanderdekroon i.p.v het vrije veld 'statustoelichting' gebruiken we nu de omschrijving van het statustype. Op deze manier maakt het niet uit wat gemeenten zelf invullen omdat we gewoon de vaste waarde gebruiken om de huidige status aan te geven. 

Dus niet:
<img width="848" alt="Scherm­afbeelding 2024-01-23 om 08 23 51" src="https://github.com/OpenWebconcept/plugin-owc-gravityforms-zaaksysteem/assets/11852816/58c206ab-2540-4d14-a087-2a2656d73a15">

Maar:
<img width="729" alt="Scherm­afbeelding 2024-01-23 om 08 24 13" src="https://github.com/OpenWebconcept/plugin-owc-gravityforms-zaaksysteem/assets/11852816/075e7395-3c05-4520-855b-a20facb6c0c7">